### DISCUSSION TITLE
Receive Data

### DISCUSSION THREAD

**Discussion by jrubiano01** [innoveit/react-native-ble-manager#1175] [Q&A]
I'm using a Huawei Band 6-09B and I managed to connect via Bluetooth, but I'm trying to get the pulse data and I don't see that information reflected in the console, I lack any method to add?, I'm using the sample App from the repository.

**Comment by enzo69006** [comment]
An example how i read data on my peripheral : 

  function readIfActivated() {
    const serviceUUID = "19b10000-e8f2-537e-4f6c-d104768a1214";
    const characteristicUUID = "19b10001-e8f2-537e-4f6c-d104768a1215";

    BleManager.read(antivolID, serviceUUID, characteristicUUID)
      .then((readData) => {
        console.log("Read data: ", readData);

        const isActivated = readData[0] === 0x01;

        setIsAntivolActivated(isActivated);

        console.log("Antivol activated state: ", isActivated ? "Activated" : "Deactivated");
      })
      .catch((error) => {
        console.log("Error reading antivol state: ", error);
      });
  }
